README.md

```diff
@@ -1,12 +1,13 @@
 # amio-sdk-js
-Server-side library implementing [Amio](https://amio.io/) API for instant messengers. It covers API calls and webhooks.
-
 [![CircleCI](https://circleci.com/gh/amio-io/amio-sdk-js.svg?style=shield)](https://circleci.com/gh/amio-io/amio-sdk-js) [![npm version](https://badge.fury.io/js/amio-sdk-js.svg)](https://badge.fury.io/js/amio-sdk-js)
 
-Let us know how to improve this library. We'll be more than happy if you report any issues or even create pull requests. ;-)
+Server-side library implementing [Amio API](https://docs.amio.io/v1.0/reference) for instant messengers. It supports API calls as well as webhooks.
+
+> Let us know how to improve this library. We'll be more than happy if you report any issues or even create pull requests. ;-)
```

Comment:
Reference `>` se mají používat pouze na reference. :) Klidně to někdy můžeme zneužít, ale vypadá to tu i dost divně. 
![image](https://user-images.githubusercontent.com/2794365/51735810-a658f680-2088-11e9-9c87-9f8b88681e9e.png)

VS

![image](https://user-images.githubusercontent.com/2794365/51735829-b40e7c00-2088-11e9-81fa-642bae0a4dbd.png)